### DISCUSSION TITLE
How to reply a quote comment?

### DISCUSSION THREAD

**Discussion by pelissarisergioo** [pwntester/octo.nvim#493] [Q&A]
I'm trying to reply to a quote comment, but I always add a new comment, does anyone know how I can do it?

**Comment by williambdean** [comment]
I don't think that this is possible at the moment. However, it would be a great feature to have if it doesn't exist. Would you be interested in digging and want to make a PR if not? 

I can imagine a `Octo comment reply` command for this to pair with the current `Octo comment suggest`
